PhoneNumberKit/UI/CountryCodePickerViewController.swift

```diff
@@ -0,0 +1,240 @@
+
+#if canImport(UIKit)
+
+import UIKit
+
+@available(iOS 11.0, *)
+protocol CountryCodePickerDelegate: class {
+    func countryCodePickerViewControllerDidPickCountry(_ country: CountryCodePickerViewController.Country)
+}
+
+@available(iOS 11.0, *)
+public class CountryCodePickerViewController: UITableViewController {
+
+    /// Common Country Codes are shown below the Current section
+    public static var commonCountryCodes: [String] = []
```

Comment:
I think this should be placed as a static property of `PhoneNumberKit` which would look like `PhoneNumberKit.CountryCodePicker.commonCountryCodes` when accessed.

This is so every static config would be in one place, thoughts?

Reply:
Agreed, I have made it so that the API is as you have described